Tutorial.md

```diff
@@ -0,0 +1,66 @@
+# MSON Tutorial
+
+MSON is short for Markdown Syntax for Object Notation, and it's a way to represent data structures in a human readable plain text form. This tutorial will take you though the basics of MSON and how you can use MSON to describe a data structure.
+
+> **NOTE:** **Additional MSON Resources**
+>
+> + [MSON Specification](https://github.com/apiaryio/mson/blob/master/MSON%20Specification.md)
+> + [Examples](https://github.com/apiaryio/mson#notational-conventions)
+
+## Data structures in MSON
+
+To get started, we're going to create a data structure in MSON to represent a Person which has a name.
+
+A data structure to store a Person could be represented in MSON using the following:
+
+```apib
+# Person
+
++ name
+```
+
+Here we've declared our own named type called "Person". This is indicated by using a heading. This is one or more `#` symbols followed by a string representing the name of our data structure. By default, named types like `Person` are object types.
+
+An object is constructed with properties, and in this case we've listed them below the heading in a list. Lists are created by putting each list item on its own line and preceding each line with either a `+`, `*` or `-`.
+
+In our `Person` data structure, we have a single property called `name`. Similarly to our Person definition, we can declare the type that our property is, with round brackets. For example, we can delcare that our name is a string using `+ name (string)`. By default all types are string's unless specified to another type, as in our example we have ommitted the string type definition.
```

Comment:
"By default all types are string's unless specified to another type" is not technically correct – for types with nested members the default is object. Not sure if we shouldn't keep it as it is here for the sake of simplicity.